app/views/users/show.html.erb

```diff
@@ -1,10 +1,35 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  </head>
-
-  <body>
-    <h1>Here is a specific user.</h1>
-  </body>
-</html>
+<div class="user_show">
+
+  <div class="user_item">
+    <div class="user_img">
+      <img src=<%= @user.photo %> alt="user_img">
+    </div>
+    <div class="user_desc">
+      <h2 class="user_name"><%= @user.name %></h2>
+      <p class="user_post_number">Number of posts: <%= @user.posts_count %></p>
+    </div>
+  </div>
+
+  <div class="user_bio">
+    <h4>Bio</h4>
+    <p class="bio_detail"><%= @user.bio %></p>
+  </div>
+
+  <ul class="post_list">
+    <% @recent_posts.each do |post| %>
+      <li class="post_item">
+        <h4><%= post.title %></h4>
+        <p class="post_text"><%= post.text %></p>
+        <p class="comments_likes">
+          Comments: <%= post.comments_count %>,
+          Likes: <%= post.likes_count %>
+        </p>
+      </li>
+    <% end %>
+  </ul>
```

Comment:
- [ ] Please match your layout to given wireframe as per the requirement:


yours|given
---|---
![image](https://github.com/binyamolango/Blog-App/assets/91486791/3e7d5418-93ac-40ff-b9f5-5d55fdea2e30)|![blog_user_page](https://github.com/binyamolango/Blog-App/assets/91486791/c5127a94-687e-4d3b-b5bd-bcafbdaf0f3c)